### PR TITLE
Add description for flapping config entries

### PR DESCRIPTION
**Pull request by Gilbert-Wong**

### CODE REVIEW

File: source/config.rst [hunk 1]
```diff
@@ -27,11 +27,11 @@ EMQ 配置变更历史
 
 为方便用户与插件开发者使用，*EMQ* 配置文件经过四次调整。
 
-1. EMQ 1.x 版本采用 Erlang 原生配置文件格式 etc/emqttd.config:
+1. EMQ 1.x 版本采用 Erlang 原生配置文件格式 etc/emqx.config:
```
Comment (turtleDeng) [inline source/config.rst]: 1.x emitted

File: source/config.rst [hunk 2]
```diff
@@ -27,11 +27,11 @@ EMQ 配置变更历史
...
 
 .. code-block:: erlang
 
-    {emqttd, [
+    {emqx, [
```
Comment (turtleDeng) [inline source/config.rst]: emqttd

File: source/config.rst [hunk 3]
```diff
@@ -64,22 +64,22 @@ Erlang 的原生配置格式多层级嵌套，对非 Erlang 开发者的用户
...
     ## Node name
-    node.name = emqttd@127.0.0.1
+    node.name = emqx@127.0.0.1
     ...
     ## Max ClientId Length Allowed.
     mqtt.max_clientid_len = 1024
     ...
 
-4. EMQ 3.0-beta1 测试版正式更名 emqttd 为 emqx ，配置名称与配置信息进行相关变化:
+4. EMQ 3.0-beta1 测试版正式更名 emqx 为 emqx ，配置名称与配置信息进行相关变化:
```
Comment (turtleDeng) [inline source/config.rst]: emqttd

Comment (Gilbert-Wong) [reply source/config.rst]: Fine, you're right

File: source/config.rst [hunk 4]
```diff
@@ -75,11 +75,11 @@ Erlang 的原生配置格式多层级嵌套，对非 Erlang 开发者的用户
 .. code-block:: properties
 
     ## Profile
-    etc/emqttd.config  ==》 etc/emqx.config
+    etc/emqx.config  ==》 etc/emqx.config
 
     ## Node name
     原先:
-    node.name = emqttd@127.0.0.1
+    node.name = emqx@127.0.0.1
```
Comment (turtleDeng) [inline source/config.rst]: emqttd

**Pull request merged**